aerogear/ionic-showcase
# feat: add support for PWA

https://issues.jboss.org/browse/INTLY-3566

This PR fixes 2 things:

1. Add support for PWA. Turns out we don't have it yet and the app won't work if the device is set to offline.
2. Remove the warning about circular dependencies between the auth service module and the sync service module.

To verify:

1. Run the app using `ionic serve --prod`
2. Login with `test/123` and try view and create some tasks (You can do this on a mobile device as well).
3. Use the network tab to turn offline (or put the app into airplane mode).
4. You should still be able to use the app and continue creating and viewing tasks.
5. Bring the device online again and you should see the data is get synced.
